### PR TITLE
docs(en-US): integrate with logto user data

### PR DESCRIPTION
**Pull request by wangsijie**
Integrate with logto user data
截屏2022-06-30 下午2 32 09

### CODE REVIEW

File: docs/docs/recipes/manage-users/on-storage.md [hunk 1]
```diff
@@ -11,11 +11,39 @@ TODO: `user.id`
...
-- pros and cons
+Traditionally, developers will create a user table in a SQL database containing the profile and other user-related information, then create a many-to-one relation in other tables, a `user` column pointing to the user table. For other identity providers, take Google as an example. Create a column named `google_id` to achieve "sign in with Google" linking. Now, for Logto, you may want to create a column `logto_id`, but it is not a good way since Logto is more than a social provider, it is a solution that can take charge of all user-related staff.
+
+With the help of Logto, **you don't need a user table**.
+
+Here is a common practice (with Logto):
+
+Forget the user table, store user info in Logto:
+
+1. Create `user_id` column in other tables, save Logto's `user.id`, refer to the previous chapter [Identify User](#identify-user).
```
Comment (simeng-li) [inline docs/docs/recipes/manage-users/on-storage.md:24]: `other tables` sounds opaque. Maybe some thing like `parent tables` or `all your reference tables`?

**Review by simeng-li — APPROVED** [review]
LGTM

File: docs/docs/recipes/manage-users/on-storage.md [hunk 2]
```diff
@@ -11,11 +11,39 @@ TODO: `user.id`
 
 ## Integrate with Logto user data
```
[the remark this answers is not in the document]
Comment (wangsijie) [reply docs/docs/recipes/manage-users/on-storage.md:12]: cool!

File: docs/docs/recipes/manage-users/on-storage.md (hunk 1 above)
Comment (wangsijie) [reply docs/docs/recipes/manage-users/on-storage.md:24]: But is not a "reference" between 2 tables, hard to describe, `parent tables` and `reference tables` is not precise, I'll merge for now, cannot figure out a name.

**Pull request merged**